extension-config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
to not mess around with conflicts and such, we normally do not change these versions in PR's unless we are ready to release. Let's say a certain @aaron-martin is also trying to change this to 2.1.1 in #4 , so you guys are now fighting over a version :P